app/src/main/res/layout/sheet_fire_clear_data.xml

```diff
@@ -48,7 +48,7 @@
         android:layout_width="0dp"
         android:layout_height="wrap_content"
         android:background="?attr/selectableItemBackground"
-        android:drawableStart="@drawable/ic_clear_browsnish_24dp"
+        android:drawableStart="@drawable/ic_cancel_24dp"
```

Comment:
I like the rename however let's keep the color info in the name as the vector has a hardcoded color in it. Let's call it `ic_cancel_brownish_24dp`